### PR TITLE
Fixing webapp to work again

### PR DESCRIPTION
**Pull request by Andrew-Ham**
Set up UID in AuthContext to pass UID into STRIPE CHECKOUT

The UID state (user's UID which is MongoDB ID) is now in the global context can be accessed everywhere in the frontend. This means the UID can be passed into the Stripe checkout endpoint that was created by the backend.

### CODE REVIEW

File: packages/frontend/src/routes/NewForm.jsx [hunk 1]
```diff
@@ -40,6 +41,7 @@ const Create = () => {
     const player = { firstName, lastName, email, paymentType, eventId };
     console.log(player);
     setIsPending(true);
+    console.log("first axios");
```
Comment (JC4github) [inline packages/frontend/src/routes/NewForm.jsx]: we can delete this

File: packages/frontend/src/routes/NewForm.jsx [hunk 2]
```diff
@@ -52,6 +54,18 @@ const Create = () => {
       console.log(error);
       setIsPending(false);
     }
+    console.log("second axios");
```
Comment (JC4github) [inline packages/frontend/src/routes/NewForm.jsx]: and this

**Review by JC4github — APPROVED** [review]
Looks good, just need to delete some console logs

File: packages/frontend/src/routes/NewForm.jsx (hunk 2 above)
Comment (Andrew-Ham) [reply packages/frontend/src/routes/NewForm.jsx]: Deleted console.log

File: packages/frontend/src/routes/NewForm.jsx (hunk 1 above)
Comment (Andrew-Ham) [reply packages/frontend/src/routes/NewForm.jsx]: Deleted console.log

**Pull request merged**